### DISCUSSION TITLE
table 的多列sort, 能提供受控模式吗?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#42542] [Q&A]
https://codesandbox.io/s/ke-kong-de-shai-xuan-he-pai-xu-antd-5-5-1-forked-d4eytf?file=/demo.tsx

**Comment by zombieJ** [comment]
这个？

https://ant.design/components/table-cn#components-table-demo-multiple-sorter

**Reply by vaynevayne** [reply]
这个不是受控的,没办法回显
